app/scenes/Settings/components/WebhookSubscriptionListItem.tsx

```diff
@@ -1,42 +1,17 @@
-import { EditIcon, TrashIcon } from "outline-icons";
 import * as React from "react";
 import { useTranslation } from "react-i18next";
 import styled from "styled-components";
 import WebhookSubscription from "~/models/WebhookSubscription";
 import Badge from "~/components/Badge";
-import Button from "~/components/Button";
 import ListItem from "~/components/List/Item";
-import Modal from "~/components/Modal";
-import useBoolean from "~/hooks/useBoolean";
-import useStores from "~/hooks/useStores";
-import WebhookSubscriptionRevokeDialog from "./WebhookSubscriptionDeleteDialog";
-import WebhookSubscriptionEdit from "./WebhookSubscriptionEdit";
+import WebhookMenu from "~/menus/WebhookMenu";
 
 type Props = {
   webhook: WebhookSubscription;
 };
 
 const WebhookSubscriptionListItem = ({ webhook }: Props) => {
   const { t } = useTranslation();
-  const { dialogs } = useStores();
-  const [
-    editModalOpen,
-    handleEditModalOpen,
-    handleEditModalClose,
-  ] = useBoolean();
-
-  const showDeletionConfirmation = React.useCallback(() => {
-    dialogs.openModal({
```

Comment:
I'd use this style of triggering dialogs in the webhook menu too, it avoids mounting them into the React tree at all on render until triggered. I'm trying to move everything over to it.